src/components/GamificationComponent/StatsCardSection.js

```diff
@@ -58,15 +58,16 @@ const useStyles = makeStyles((theme) => ({
     display: "flex",
     marginTop: theme.spacing(4),
     marginBottom: theme.spacing(4),
-    [theme.breakpoints.down("sm")]: {
+    [theme.breakpoints.down("755")]: {
       flexWrap: "wrap",
     },
   },
   mediaCard: {
     width: 300,
+    minWidth: 165,
     boxShadow: "0px 6px 12px -6px rgba(24, 39, 75, 0.12), 0px 8px 24px -4px rgba(24, 39, 75, 0.08)",
     marginRight: theme.spacing(4),
-    [theme.breakpoints.down("sm")]: {
+    [theme.breakpoints.down("755")]: {
```

Comment:
Is setting minWidth not enough to give minimum possible width for content to not get cropped?

The breakpoint is hardcoded, just not looking good to me. Is there any other workaround?